packages/plugins/components/src/NavBar/navbar.stories.tsx

```diff
@@ -9,7 +9,7 @@ export default {
   component: NavBar,
 } as ComponentMeta<typeof NavBar>;
 
-const useStyles = makeStyles((theme: Theme) => ({
+const useStyles = makeStyles((_theme: Theme) => ({
```

Comment:
just wonder what is the difference? I saw in the code sometimes we use theme and sometimes use _theme

Reply:
'_' prefix let compiler now that even though this parameter is not in use at tis point, we want to keep it named in code. Otherwise eslint will complain about unused parameter.

None of used variables should have `_` prefix. If variable is not used we can either delete it or add _ prefix.
I decided to add prefix here, as I know that I will use it in my next PR :)